### ISSUE TITLE
Does not respond anything with :Denite

### ISSUE THREAD

**Issue by aiya000** [akiyosi/gonvim#7]
Hi :smile:  
Thank you for present this great app!

I have a basic problem,
that gonvim perfectly hanged up after `:Denite foo`.

There is a minimum procedure for reproducing :point_down: 

```console
$ cat local.vim
set rtp+=~/.vim/bundle/repos/github.com/Shougo/denite.nvim

$ gonvim -u local.vim  # and do `:Denite buffer`, `:Denite file`, or other denite.nvim command
Unhandle event option_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
Unhandle event default_colors_set
Unhandle event option_set
Unhandle event mode_info_set
^C
```

Do you have some knowledge for this? :thinking:  
Please help me :dog2:

**Comment by akiyosi** [comment]
Hi,
Thanks for the report.

I fixed this issue and updated the release [0.3.0-dev](https://github.com/akiyosi/gonvim/releases/tag/0.3.0-dev).
Please try `:Denite foo` operation in your environment.

**Comment by aiya000** [comment]
I confirmed that fixes, thanks! :smile:
